components/WorkInProgress.tsx

```diff
@@ -0,0 +1,35 @@
+import Alert from 'react-bootstrap/Alert'
+
+import { useTranslation } from 'next-i18next'
+import { Trans } from 'react-i18next'
+import React, { useState } from 'react'
+
+export const WorkInProgress: React.FC = () => {
+  const { t } = useTranslation('common')
+  const [show, setShow] = useState(true)
+
+  if (show) {
+    return (
+      <Alert variant="primary" onClose={() => setShow(false)} dismissible fixed-top="true">
+        <div className="work-in-progress">
+          <div className="work-in-progress-content">
+            <span className="work-in-progress-title">{t('work-in-progress.title')}</span>
+            &nbsp;
+            <span className="work-in-progress-message">
+              {
+                // TODO correct link
+                <Trans i18nKey="work-in-progress.message">
+                  While we test this new part of UI Online, the information here may be limited for your claim.
+                  We&apos;re working to add information about common issues that lead to&nbsp;
+                  <a href="https://navapbc.com">pending payments</a>
+                </Trans>
```

Comment:
React learning question: Why do we have text here and also have text in public/locales/en/common.json?

Reply:
Its how the Trans component works! https://react.i18next.com/latest/trans-component

it does some parsing of the default language so that you can reference the links in other languages

Reply:
its got strong ReGex Magic vibes, tbh.